src/components/Person/Person.spec.jsx

```diff
@@ -1,72 +1,72 @@
-import { mount } from '@cypress/react18';
-import { Person } from './Person';
+import { mount } from "@cypress/react18";
+import { Person } from "./Person";
 
-describe('Person', () => {
-  describe('', () => {
+describe("Person", () => {
+  describe("", () => {
     beforeEach(() => {
       const misha = {
-        name: 'Misha',
+        name: "Misha",
         age: 37,
-        sex: 'm',
+        sex: "m",
         isMarried: true,
-        partnerName: 'Natasha',
+        partnerName: "Natasha",
       };
 
       mount(<Person person={misha} />);
     });
 
-    it('should print a name', () => {
-      cy.get('.Person__name').should('have.text', 'My name is Misha');
+    it("should print a name", () => {
+      cy.get(".Person__name").should("have.text", "My name is Misha");
     });
 
-    it('should print an age', () => {
-      cy.get('.Person__age').should('have.text', 'I am 37');
+    it("should print an age", () => {
+      cy.get(".Person__age").should("have.text", "I am 37");
     });
 
-    it('should print a partner', () => {
-      cy.get('.Person__partner').should('have.text', 'Natasha is my wife');
+    it("should print a partner", () => {
+      cy.get(".Person__partner").should("have.text", "Natasha is my wife");
     });
   });
 
-  describe('', () => {
-    it('should show partner as a husband when the person is female', () => {
+  describe("", () => {
+    it("should show partner as a husband when the person is female", () => {
       const olya = {
-        name: 'Olya',
-        sex: 'f',
+        name: "Olya",
+        sex: "f",
         age: 38,
         isMarried: true,
-        partnerName: 'Maksym',
+        partnerName: "Maksym",
       };
 
       mount(<Person person={olya} />);
 
-      cy.get('.Person__partner').should('have.text', 'Maksym is my husband');
+      cy.get(".Person__partner").should("have.text", "Maksym is my husband");
     });
 
-    it('should show correct message if a person is not married', () => {
+    it("should show correct message if a person is not married", () => {
       const tanya = {
-        name: 'Tanya',
-        sex: 'f',
+        name: "Tanya",
+        sex: "f",
         age: 28,
         isMarried: false,
-        partnerName: 'Some Partner',
+        partnerName: "Some Partner",
```

Comment:
The `partnerName` property is included in the `tanya` object even though `isMarried` is false. This might not affect the test outcome, but it's unnecessary and could be removed for clarity.